docs/_index.md

```diff
@@ -20,9 +20,15 @@ sitemap:
 
 <div id="board-carousel" class="carousel max-page">
   <ul tabindex="0">
+  <li id="c0_slide1">
```

Comment:
Fix numbering and add this on lines 116 and following as well otherwise it breaks the button functionality

Please also test fully that a 0 index on these works

Reply:
done